src/strings/csv/en.csv

```diff
@@ -311,6 +313,7 @@ ENTER_SIX_DIGIT_KEY,Enter 6-digit key
 ENVIRONMENTAL_NOTES,Environmental Notes
 ENVIRONMENTAL_NOTES_PLACEHOLDER,"Important notes about landscape, climate, and environmental conditions"
 EST_READY_DATE,Est. Ready Date
+EST_TOTAL_PLANTS_PLANTING_DENSITY_AREA,Est. Total Plants (Planting Density × Area)
```

Comment:
not sure if translation will know what `Est.` is, should we expand this fully `Estimated`? or add a hint after a comma
`Est. Total Plants (Planting Density x Area), Estimated Total Plants`